Makefile

```diff
@@ -42,9 +42,9 @@ CONTAINER_RUNTIME_COMMAND := $(or ${CONTAINER_COMMAND}, ${CONTAINER_RUNTIME_COMM
 LINT_CODE_STYLING_DIRS := src/tests src/triggers src/assisted_test_infra/test_infra src/assisted_test_infra/download_logs src/service_client src/consts src/virsh_cleanup src/cli
 
 # assisted-service
-SERVICE := $(or $(SERVICE), quay.io/edge-infrastructure/assisted-service:latest)
+SERVICE := $(or $(SERVICE), quay.io/edge-infrastructure/assisted-service:ocm-2.7)
```

Comment:
I wonder if we can make some adjustments that don't demand us to handle all those tag changes for the branches
I mean, if anything I would assume we override ``SERVICE`` somewhere to use the right service image, and not use the default. Is it not the case?

Reply:
You're right, I'll see if I can do it from release repo

Reply:
and yes SERVICE shouldn't be required

Reply:
you know what, maybe this change is good for easing local testing of this version